content/docs/guide.md

```diff
@@ -1,94 +1,275 @@
+Each job represents a feature of Packit. All the jobs are defined under the `jobs` key.
+For each job you need to specify the following information:
+* `job`: the type of the job, the feature you want Packit to do.
+* `trigger`: when the job is run (`pull_request`, `commit` or `release`).
+* additional information needed for the job (like what targets to build, for what branch to run the job,...)
+
+(Note that for each job, not all the triggers are supported.)
+
+As an example you can take a look at the following config snippet
+(these are also the default jobs used if you don't set the `jobs` key):
+```yaml
+jobs:
+- job: copr_build
+  trigger: pull_request
+  targets: [fedora-stable]
+
+- job: tests
+  trigger: pull_request
+  targets: [fedora-stable]
+
+- job: propose_downstream
+  trigger: release
+  dist_git_branches:
+    - fedora-all
+```
+
+Most of the attributes can be defined both on global level (outside of the `jobs` key)
+or for a single job only (like e.g. `targets` in the snippet above).
+More about this overriding mechanism can be found [here](/docs/configuration/#overriding-global-parameters).
+Another useful feature are [aliases](/docs/configuration/#aliases) used to avoid
+hardcoded values that changes when there is a new distribution release.
+
+
+### Available jobs
+
+* [`copr_build`](/docs/configuration/#copr_build): An RPM build triggered for pull-requests, new branch commits or releases.
+  (Can be used to verify that package is buildable,
+  to easily install package including the proposed change
+  or to provide long-term Copr repositories.)
+* [`tests`](/docs/configuration/#tests): Test suit using TMT/FMF definition run in the [Testing Farm](TBD)
+  (Can be used as a next step to Copr build or without build at all.)
+* [`production_build`](/docs/configuration/#production_build): A scratch Koji build triggered for the upstream state of project.
+* [`propose_downstream`](/docs/configuration/#propose_downstream): For upstream release, Packit prepares a Fedora release.
+  (Source is saved to the Lookaside Cache and a dist-git pull-request is created for each configured branch.)
+* [`koji_build`](/docs/configuration/#koji_build): A downstream Koji build triggered when there is a new dist-git commit in a given branch.
+* [`bodhi_update`](/docs/configuration/#bodhi_update): A Bodhi update created for a successfully finished Koji build.
+
+
+More about the jobs and how to tweak them can be found on a [configuration page](/docs/configuration#jobs).
+
+
+### Actions
+
+Actions are a way how to redefine or tweak the Packit's workflows.
+These are user-defined commands that can replace a part of Packit's job (e.g. to get version, or to create a tarball)
+or are run in a particular step as a hook (with no action run by default).
+
+Take a look at the tables [here](/docs/actions/) to know what actions are available for each job.
+
+
+### How to try that for real
```

Comment:
This was aimed to verify the build went well locally and that there is a high chance it will work in the service. At least, that was the goal. (I don't want to focus on using CLI for actual usage, just as a way how to reproduce/debug problems...)

I am still not satisfied with that -- at least, there is that duplicate `packit init` output...